woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -223,6 +223,42 @@ def test_accessor_getattr(sample_df):
         sample_df.ww.not_present
 
 
+def test_getitem(sample_df):
+    xfail_koalas(sample_df)
+    df = sample_df.copy()
+    df.ww.init(time_index='signup_date', index='id', name='dt_name')
+    assert list(df.columns) == list(df.ww.schema.columns)
+
+    subset = ['id', 'email', 'signup_date']
+    subset_ww = df.ww[subset].ww
```

Comment:
It's probably worth checking that the contents of the subset DataFrames and Series' are as expected in addition to the Schema checks, so we might as well get `subset_df = df.ww[subset]` so we can confirm `subset_df` is the same as `df[subset]`.

Plus then we can access the index and whatnot with `subset_df.ww.index`

Reply:
✅